src/Wsdl/Event/RequestFailEvent.php

```diff
@@ -0,0 +1,153 @@
+<?php
+declare(strict_types = 1);
+
+namespace Skaut\Skautis\Wsdl\Event;
+
+use Serializable;
+use Throwable;
+
+class RequestFailEvent implements Serializable
+{
+
+    /**
+     * @var string Nazev funkce volane pomoci SOAP requestu
+     */
+    private $fname;
+
+    /**
+     * Parametry SOAP requestu na server
+     *
+     * @var array<int, mixed>
+     */
+    private $args;
+
+    /**
+     * @var float Doba trvani pozadvku
+     */
+    private $time;
+
+    /**
+     * Ne vsechny exceptions jdou serializovat.
+     * Po unserializaci je null.
+     *
+     * @var Throwable|null
+     */
+    private $throwable;
+
+    /**
+     * Po unserializaci je zde trida excpetion
+     *
+     * @var string
+     */
+    private $exceptionClass = '';
```

Comment:
Tady není potřeba mít výchozí hodnotu, protože [v konstruktoru](https://github.com/skaut/Skautis/pull/96/files#diff-08e952434ddd2e43419633cd6768226aR66) ji vždy přepíšeš + ten komentář IMO není pravda, protože je to inicializováno vždy.